### DISCUSSION TITLE
Sendmail support

### DISCUSSION THREAD

**Discussion by marcelschmidtdev** [dani-garcia/vaultwarden#2198] [Ideas]
Hello, I'd like to ask how complex it would be to add support for sendmail?
I saw that the email lib you are using [is supporting it](https://docs.rs/lettre/latest/lettre/sendmail/struct.SendmailTransport.html) but unfortunately I'm not familiar with Rust and can't implement it myself.
Thanks in advance!

**Comment by BlackDex** [comment]
Hmm, using sendmail also means we should add sendmail to the images. Which is something which i would rather not do.

May i ask what the reason is? What is it you missing right now?

**Reply by marcelschmidtdev** [reply]
My plan was to bind sendmail to the Docker container to be able to use my local mail server without having to add an SMTP user only for Vaultwarden.

**Comment by soruh** [comment]
I have opened #3147 implement this.

**Reply by soruh** [reply]
As mentioned in the PR. Sendmail support does not require bundling sendmail but rather allows using an already installed sendmail.
